inc/inventory/asset/inventoryasset.class.php

```diff
@@ -345,10 +345,16 @@ protected function addOrMoveItem(array $input): void {
          'items_id' => $input['items_id']
       ]);
 
-      if (isset($citem->fields['id'])) {
-         $citem->delete(['id' => $citem->fields['id']], true, $this->withHistory());
+      $itemtype = $input['itemtype'];
+      $item = new $itemtype;
+      $item->getFromDb($input['items_id']);
+
+      if (($item->fields['is_global'] ?? 0) == 0) {
```

Comment:
`is_global` is a tinyint so it should be casted to `true`/`false` by the mysli dirver.
```suggestion
      if (!($item->fields['is_global'] ?? false)) {
```